packages/smooth_app/lib/pages/preferences/user_preferences_page.dart

```diff
@@ -165,6 +167,46 @@ class _UserPreferencesPageState extends State<UserPreferencesPage>
               ),
             ),
           ),
+          SliverToBoxAdapter(
+            child: Column(
+              children: <Widget>[
+                getUserPreferences(
+                  type: PreferencePageType.ACCOUNT,
+                  userPreferences: userPreferences,
+                ).getOnlyHeader(),
+                ElevatedButton(
```

Comment:
Please use a "smooth" button instead.

Reply:
I tried using "smooth" button but it was not looking good so that's I removed it.

Reply:
and there is no icon required for this.

Reply:
ok